### PR TITLE
VIH-10125 Upgrade to .NET 8

### PR DESCRIPTION
**Pull request by oliver-scott**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/VIH-10125

### CODE REVIEW

File: charts/vh-video-web/values.dev.template.yaml [hunk 1]
```diff
@@ -10,4 +10,5 @@ java:
         EJUDAD__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
         EJUDAD__REDIRECTURI: https://${SERVICE_FQDN}/home
         DOM1__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
-        DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
\ No newline at end of file
+        DOM1__REDIRECTURI: https://${SERVICE_FQDN}/home
+        VHSERVICES__VIDEOAPIURL: https://vh-video-api-pr-643.dev.platform.hmcts.net
```
Comment (oliver-scott) [inline charts/vh-video-web/values.dev.template.yaml:14]: TODO revert before merge

**Review by shaed-parkar — APPROVED** [review]

**Review by ronaldmalone — APPROVED** [review]

**Review by himesh99 — APPROVED** [review]
lgtm

**Pull request merged**